broker/src/main/java/io/camunda/zeebe/broker/clustering/ClusterServices.java

```diff
@@ -7,47 +7,42 @@
  */
 package io.camunda.zeebe.broker.clustering;
 
+import io.atomix.cluster.AtomixCluster;
 import io.atomix.cluster.ClusterMembershipService;
 import io.atomix.cluster.messaging.ClusterCommunicationService;
 import io.atomix.cluster.messaging.ClusterEventService;
 import io.atomix.cluster.messaging.MessagingService;
-import io.atomix.core.Atomix;
-import io.atomix.primitive.partition.ManagedPartitionGroup;
 import java.util.concurrent.CompletableFuture;
 
-public final class ClusterServices {
+public class ClusterServices {
```

Comment:
:thought_balloon: I can imagine making this an interface later, and having the `AtomixCluster` based adapter for it. I would do it now personally, as it's little effort and I think it has some obvious benefits for testing, but I'm happy to leave that for later.